spec/support/controller_exceptions.rb

```diff
@@ -0,0 +1,23 @@
+module ControllerExceptions; end
+
+# Test group helpers for allowing exceptions from controller tests to be caught by specs.
+module ControllerExceptions::TestGroupHelpers
+  BYPASS_RESCUE_CONSTANT_NAME = :__controller_exceptions_test_group_helpers_bypass_rescue
+
+  def self.extended(module_)
+    module_.module_eval do
+      let(BYPASS_RESCUE_CONSTANT_NAME) { true }
+      before(:each) do
+        bypass_rescue if send(BYPASS_RESCUE_CONSTANT_NAME)
```

Comment:
that should be part of the feature spec. you would raise an exception in more places than you catch it (hopefully)

if you catch an exception in more places than you raise it then... the program is not structured correctly.